packages/babel-preset-expo/src/index.ts

```diff
@@ -112,6 +112,10 @@ function babelPresetExpo(api: ConfigAPI, options: BabelPresetExpoOptions = {}):
     extraPlugins.push(expoRouterBabelPlugin);
   }
 
+  if (api.caller((caller) => caller?.isServer)) {
+    extraPlugins.push(require('@babel/plugin-transform-dotall-regex'));
```

Comment:
Doesn't Node LTS support this in V8?

Reply:
Moving this change to another PR so I can evaluate it further.